### ISSUE TITLE
Documentation doesn't match code for lmtp_downstream targets

### ISSUE THREAD

**Issue by smlx** [foxcpp/maddy#238]
# Describe the bug

Documentation specifies `target unix:///...` inside `lmtp_downstream` [here](https://github.com/foxcpp/maddy/blob/master/docs/tutorials/smtp-only.md#dovecot), but maddy complains:

```
unexpected directive: target
```

Taking a look at the code I figured out that `targets` works.

I'm not sure if the documentation needs updating, or the code :smile: 

This may also affect `smtp_downstream` since the parsing code looks to be shared.

# Steps to reproduce

Put a line like this inside a `lmtp_downstream` block:

```
target unix:///foo/bar.sock
```

# Log files

```
/etc/maddy.conf:26: unexpected directive: target	
```

# Configuration file

n/a

# Environment information

* maddy version: 0.3.1

**Comment by foxcpp** [comment]
Woops, good find. Documentation needs updating, directive is called "targets" since it allows to specify multiple values.

**Comment by foxcpp** [comment]
Oh, the man page also has the same typo. I feel silly. :)

**Comment by foxcpp** [comment]
Fixed both in 2a19d455f38b5d7fdf97915c8423f87c0f1393d5.
